src/arrayMethodSort.js

```diff
@@ -4,8 +4,21 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (a, b) =>
+    String(a) > String(b)
+  ) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(String(this[i]), String(this[j])) > 0) {
```

Comment:
```suggestion
        if (compareFunction(this[i], this[j]) > 0) {
``` 
Тут нет необходимости приводить к строкам, мне кажется